### PR TITLE
Name AppVeyor Tags even more aggressively

### PR DESCRIPTION
**Pull request by Vogel612**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -107,7 +107,7 @@ artifacts:
     name: Rubberduck
 deploy:
   - provider: GitHub
-    tag: Rubberduck Stable v$(appveyor_build_version)
+    tag: RETAGMEWITHAMESSAGE
```
Comment (bclothier) [inline appveyor.yml]: i feel that this needs a bit explanation - is this a special code to AV to retag it using the commit message? or is it just a comment to us meatbags to retag it?

File: appveyor.yml [hunk 2]
```diff
@@ -118,11 +118,11 @@ deploy:
     on: 
       branch: master
   - provider: GitHub
-    tag: Rubberduck v$(appveyor_build_version)
+    tag: Prerelease-v$(appveyor_build_version)
     release: $(release_name)
     draft: false
     prerelease: true
-    description: "AppVeyor build on [$(appveyor_repo_branch)] - https://ci.appveyor.com/project/rubberduck-vba/rubberduck/build/$(appveyor_build_version)<br/><br/>Merges #$(appveyor_pull_request_number) - $(appveyor_pull_request_title)"
+    description: "AppVeyor build on [$(appveyor_repo_branch)] - https://ci.appveyor.com/project/rubberduck-vba/rubberduck/build/$(appveyor_build_version)"
```
Comment (bclothier) [inline appveyor.yml]: Bummer this isn't working. Would have been useful to see what's what just browsing the releases.

File: appveyor.yml (hunk 1 above)
Comment (Vogel612) [reply appveyor.yml]: The latter

File: appveyor.yml (hunk 2 above)
Comment (Vogel612) [reply appveyor.yml]: Ya.that was the idea

**Pull request merged**